drivers/usb/device/usb_dc_dw.c

```diff
@@ -71,48 +71,40 @@ static inline void _usb_dw_int_unmask(void)
 #endif
 }
 
-#if (CONFIG_SYS_LOG_USB_DRIVER_LEVEL >= SYS_LOG_LEVEL_DEBUG)
+#if (LOG_LEVEL >= LOG_LEVEL_DEBUG)
```

Comment:
you can remove this if def. if debug level is off compiler will remove all debug messages and all code around it.

Reply:
Removed